### PR TITLE
Bump to 1.19.1, various fixes

### PR DESCRIPTION
**Pull request by RaptaG**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,11 +1,11 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties from https://fabricmc.net/develop
-minecraft_version=1.19
+minecraft_version=1.19.1
 yarn_mappings=1.19+build.4
```
Comment (theRookieCoder) [inline gradle.properties]: `1.19.1+build.3`

**Review by theRookieCoder — APPROVED** [review]

**Comment by theRookieCoder** [comment]
LGTM

File: gradle.properties (hunk 1 above)
Comment (RaptaG) [reply gradle.properties]: Resolved, but I want to add one more thing so don’t merge it just yet

**Comment by RaptaG** [comment]
I need to finish that..

File: gradle.properties [hunk 2]
```diff
@@ -1,11 +1,11 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties from https://fabricmc.net/develop
-minecraft_version=1.19
-yarn_mappings=1.19+build.4
+minecraft_version=1.19.1
+yarn_mappings=1.19.1+build.3
```
Comment (theRookieCoder) [inline gradle.properties]: Latest version is `1.19.1+build.5`

File: src/main/resources/fabric.mod.json [hunk 3]
```diff
@@ -4,19 +4,16 @@
   "version": "${version}",
   "name": "StairAutoJump",
   "description": "Turns on auto jump when in/on stairs so that you can move faster!",
-  "authors": ["theRookieCoder", "RaptaG"],
```
Comment (theRookieCoder) [inline src/main/resources/fabric.mod.json:7]: Why did you remove yourself?

**Review by theRookieCoder — CHANGES_REQUESTED** [review]

File: src/main/resources/fabric.mod.json (hunk 3 above)
Comment (RaptaG) [reply src/main/resources/fabric.mod.json:7]: I didn't actually, I just made it clear that I'm not the author :)
https://github.com/theRookieCoder/StairAutoJump/pull/13/files#diff-882c6606823cf6c4e4114c1125850a04ed3180c48aef0e1467f1e4022b235db2R14

File: gradle.properties (hunk 2 above)
Comment (RaptaG) [reply gradle.properties]: https://github.com/RaptaG/StairAutoJump/commit/93e5de93ff888105ab5b26b7c7fa9bea068b346a

**Comment by RaptaG** [comment]
Can't wait for #11 and #12 to be fixed btw

File: src/main/resources/fabric.mod.json (hunk 3 above)
Comment (theRookieCoder) [reply src/main/resources/fabric.mod.json:7]: Ahh I see

**Comment by theRookieCoder** [comment]
Alright looks good, thank you for the help!

**Review by theRookieCoder — APPROVED** [review]

**Comment by RaptaG** [comment]
> Alright looks good, thank you for the help!

Glad to be a part of this project!
